reactor-core/src/main/java/reactor/core/Scannable.java

```diff
@@ -380,28 +395,77 @@ default String name() {
 	}
 
 	/**
-	 * Check this {@link Scannable}e and its {@link #parents()} for a name an return the
-	 * first one that is reachable.
-	 *
-	 * @return the name of the first parent that has one defined (including this scannable)
+	 * @return the default operatorName of this operator
 	 */
 	default String operatorName() {
```

Comment:
It's already in 3.1 but having now operatorChain made me realize that we could over time associate more and more Scannable with operators. I think this one is fine, the documentation might not be tho and should say: "Will format the name of this scannable accordingly if operator implementation, ex: ...."

Reply:
How about `stepName()` and `steps()`? `name()` would still represent an upstream search of a user-provided name for the sequence.